core/src/main/scala/akka/kafka/internal/SingleSourceLogic.scala

```diff
@@ -56,12 +59,15 @@ private[kafka] abstract class SingleSourceLogic[K, V, Msg](
     self.watch(consumer)
 
     val partitionAssignedCB = getAsyncCallback[Set[TopicPartition]] { newTps =>
+      // Is info too much here? Will be logged at startup / rebalance
```

Comment:
perhaps use info/debug depending on the wakeupDebug flag

Reply:
+1